_posts/2019-08-01-CocoaPods-1.8.0-beta.markdown

```diff
@@ -0,0 +1,147 @@
+---
+layout: post
+title:  "CocoaPods 1.8 Beta is Here!"
+author: dnkoutso
+categories: cocoapods releases
+---
+
+_CocoaPods 1.8_ switches to CDN as the default and comes with a few enhancements!
+
+<!-- more -->
+
+This is an exciting release as the [master specs repo](https://github.com/CocoaPods/Specs) is no longer required to be cloned to start using CocoaPods!
+
+### CDN as Default
```

Comment:
@igor-makarov does this all make sense?

Reply:
I think we might wanna clarify that 1.7.0 can't actually work with the current CDN, just in case.

Reply:
sounds good.

Reply:
done.